index.bs

```diff
@@ -2916,6 +2916,246 @@ value and terminate the operation.
+offers them the [=credential=] they previously deleted. The user selects that
+[=credential=]. After rejecting the sign-in attempt, the [=[WRP]=] runs:
+
+```javascript
+PublicKeyCredential.signal({
+    unknownCredential: {
+        rpId: "example.com",
+        credentialId: "aabbcc"  // credential id the user just tried, base64url
+    }
+});
+```
+
+The [=authenticator=] then deletes or hides the [=credential=] from future
+[=authentication ceremonies=].
+
+</div>
+
+#### <dfn>currentCredentials</dfn> report #### {#sctn-currentCredentialsReport}
+
+Signals the complete list of [=credential ids=] for a given user, as well as the
+user's current {{PublicKeyCredentialEntity/name}} and
+{{PublicKeyCredentialUserEntity/displayName}}.
+
+The [=currentCredentials=] [=validation algorithm=] takes a
+{{CurrentCredentialsReport}} |options| and executes these steps:
+1. Let |effectiveDomain| be the |callerOrigin|'s [=effective domain=].  If
+    [=effective domain=] is not a [=valid domain=], then throw a
+    "{{SecurityError}}" {{DOMException}}.
+1. If <code>|options|.{{CurrentCredentialsReport/rpId}}</code> [=is not a
+    registrable domain suffix of and is not equal to=] |effectiveDomain|, then
+    throw a "{{SecurityError}}" {{DOMException}}.
+
+    TODO: this may require updating once #2040 lands.
+1. If the result of [=base64url encoding | base64url decoding=]
+    <code>|options|.{{CurrentCredentialsReport/userId}}</code> is an error, then
+    throw a {{TypeError}}.
+1. If
+    <code>|options|.{{CurrentCredentialsReport/allAcceptedCredentialIds}}</code>
+    is present:
+    1. [=list/For each=] |credentialId| in
+        <code>|options|.{{CurrentCredentialsReport/allAcceptedCredentialIds}}</code>:
+       1. If the result of [=base64url encoding | base64url decoding=]
+           |credentialId| is an error, then throw a {{TypeError}}.
+
+The [=currentCredentials=] [=credential report/authenticator actions=] take a
+{{CurrentCredentialsReport}} |options| and are as follows:
+1. Let |userId| be result of [=base64url encoding | base64url decoding=]
+    <code>|options|.{{CurrentCredentialsReport/userId}}</code>.
+1. Assertion: |userId| is not an error.
+1. Let |credential| be
+    <code>[=credentials map=][|options|.{{CurrentCredentialsReport/rpId}}, |userId|]</code>.
+1. If |credential| does not exist, abort these steps.
+1. If <code>|options|.{{CurrentCredentialsReport/user}}</code> is present,
+    update the |credential|'s [=public key credential source/otherUI=] to match
+    <code>|options|.{{CurrentCredentialsReport/user}}.{{CurrentCredentialUserDetails/name}}</code>
+    and
+    <code>|options|.{{CurrentCredentialsReport/user}}.{{CurrentCredentialUserDetails/displayName}}</code>
+1. If
+    <code>|options|.{{CurrentCredentialsReport/allAcceptedCredentialIds}}</code>
+    is present and
+    <code>|options|.{{CurrentCredentialsReport/allAcceptedCredentialIds}}</code>
```

Comment:
I think this line can be omitted. :slightly_smiling_face:

```suggestion
```

Reply:
This parameter is now required.